docs/en/latest/terminology/plugin-metadata.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Plugin Metadata
+keywords:
+  - API Gateway
+  - Apache APISIX
+  - Plugin Metadata
+description: Plugin Metadata in Apache APISIX.
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Description
+
+In this document, you will learn the basic concept of plugin metadata in APISIX and why you may need them.
+
+Explore additional resources at the end of the document for more information on related topics.
+
+## Overview
+
+In APISIX, a plugin metadata object is used to configure the common metadata field(s) of all plugin instances sharing the same plugin name. It is useful when a plugin is enabled across multiple objects and requires a universal update to their metadata fields.
+
+The following diagram illustrates the concept of plugin metadata using two instances of syslog plugins on two different routes, as well as a plugin metadata object setting a global log_format for the syslog plugin:
+
+![plugin_metadata](https://static.apiseven.com/uploads/2023/04/17/Z0OFRQhV_plugin%20metadata.svg)
+
+Without otherwise specified, the `log_format` on plugin metadata object should apply the same log format uniformly to both `syslog` plugins. However, since the `syslog` plugin on the `/orders` route has a different `log_format`, requests visiting this route will generate logs in the `log_format` specified by the plugin in route.
+
+Metadata properties set at the plugin level is more granular and has a higher priority over the "global" metadata object.
+
+Plugin metadata objects should only be used for plugins that have metadata fields. For more details on which plugins have metadata fields, please refer to the plugin reference guide.
```

Comment:
Plugin reference guide? Maybe a better way to phrase it would be "please refer to their documentation" or something along those lines.

i.e., since we don't have a page that lists this, we have to ask them to refer to each plugin's documentation to see if it has metadata that can be configured.

Reply:
Thanks, I'll change it